examples/todomvc/src/template.rs

```diff
@@ -0,0 +1,54 @@
+use store::{ItemList, ItemListTrait};
+
+fn escape_html(val: String) -> String {
```

Comment:
One thing that might be neat to show off is pulling in a crate for this, and presumably one of these already exists on crates.io?